src/opnsense/www/js/opnsense_theme.js

```diff
@@ -40,70 +46,119 @@ $(document).ready(function () {
-            if ($("#navigation").hasClass("col-sidebar-left")) {
-                $("#navigation > div > nav > #mainmenu > div > div").on({
-                    mouseout: function() {$("#navigation.col-sidebar-left").css("width", "70px"); },
-                    mouseover: function() {$("#navigation.col-sidebar-left").css("width", "380px"); }
-                });
-                $("#navigation > div > nav > #mainmenu > div > a").on({
-                    mouseout: function() {$("#navigation.col-sidebar-left").css("width", "70px"); },
-                    mouseover: function() {$("#navigation.col-sidebar-left").css("width", "380px"); }
-                });
-            }
-        });
-
-        /* on resize - toggle sidebar / main navigation */
-        $(window).on('resize', function(){
-            var win = $(this);
-            if ((win.height() < 675 || win.width() < 760) && $("#navigation").not("col-sidebar-hidden")) {
-                $("#navigation").addClass("col-sidebar-hidden");
-                if ($("#navigation").hasClass("col-sidebar-left")) {
-                    opnsense_sidebar_toggle(false);
-                }
-            } else if ((win.height() >= 675 && win.width() >= 760) && $("#navigation").hasClass("col-sidebar-hidden")) {
-                $("#navigation").removeClass("col-sidebar-hidden");
-                if (window.sessionStorage && sessionStorage.getItem('toggle_sidebar_preset') == 1) {
-                    opnsense_sidebar_toggle(false);
-                }
-            }
-        });
-
-        /* only show toggle button when style is loaded */
-        $(".toggle-sidebar").show();
-
-        /* auto-collapse if previously requested */
-        if (window.sessionStorage && sessionStorage.getItem('toggle_sidebar_preset') == 1) {
-            opnsense_sidebar_toggle(false);
-        }
-    }
+	
+	/* disable mouseevents on toggle and resize */
+	function mouse_events_off() {
+		layer1_a.off(nomouse);
+		layer2_a.off(nomouse);
+		layer1_div.off(nomouse);
+		layer2_div.off(nomouse);
+								}
+	/* trigger mouseevents on startup */
+	function trigger_sidebar() {
+		layer1_a.first().trigger("mouseenter");
+		layer1_a.first().trigger("mouseleave");
+		layer1_div.removeClass("in");
+		}
+	
+	/* transition duration - time */
+	function transition_duration(time) {
+		$.fn.collapse.Constructor.TRANSITION_DURATION = time;
+		}
+	
+	function opnsense_sidebar_toggle(store) {
+		$("#navigation").toggleClass("col-sidebar-left");
+		$("main").toggleClass("col-sm-9 col-sm-push-3 col-lg-10 col-lg-push-2 col-lg-12");
+		$(".toggle-sidebar > i").toggleClass("fa-chevron-right fa-chevron-left");
+		if ($("#navigation").hasClass("col-sidebar-left")) {
+			$(".brand-logo").css("display", "none");
+			$(".brand-icon").css("display", "inline-block");
+			trigger_sidebar();
+			if (store && window.sessionStorage) {
+				sessionStorage.setItem('toggle_sidebar_preset', 1);
+				transition_duration(0);
+				}
+			} else {
```

Comment:
To answer your question: Only from the indent in this diff view - to which if does it belong and to which dies it really belong?